examples/gno.land/r/matijamarjanovic/home/home.gno

```diff
@@ -0,0 +1,230 @@
+package home
+
+import (
+	"std"
+	"strconv"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	pfp          string // link to profile picture
+	pfpCaption   string // profile picture caption
+	abtMe        string
+	currentColor string
+
+	redVotes   int64
+	blueVotes  int64
+	greenVotes int64
+
+	blueLink  string
+	redLink   string
+	greenLink string
+)
+
+func init() {
+	pfp = "https://muaythairecords.com/fighters/rodtang-jitmuangnon/ogimage"
+	pfpCaption = "My profile picture - Rodtang Jitmuangnon (Muay Thai fighter)"
+	abtMe =
+		`### About me
+Motivated Computer Science student with strong
+analytical and problem-solving skills. Proficient in
+programming and version control, with a high level of
+focus and attention to detail. Eager to apply academic
+knowledge to real-world projects and contribute to
+innovative technology solutions.
+
+In addition to my academic pursuits, I enjoy traveling and staying active through weightlifting. I have a keen interest in electronic music and often explore various genres. I believe in maintaining a balanced lifestyle that complements my professional development.`
+	redVotes = 0
+	greenVotes = 0
+	blueVotes = 0
+	currentColor = `rgb(120, 120, 120)`
+	blueLink = "https://www.google.com"
+	redLink = "https://www.google.com"
+	greenLink = "https://www.google.com"
+}
+
+func UpdatePFP(url, caption string) {
+	AssertAuthorized()
+	pfp = url
+	pfpCaption = caption
+}
+
+func UpdateAboutMe(col1 string) {
+	AssertAuthorized()
+	abtMe = col1
+}
+
+func maxOfThree(a, b, c int64) int64 {
+	max := a
+	if b > max {
+		max = b
+	}
+	if c > max {
+		max = c
+	}
+	return max
+}
+
+func calculateRGB(red, green, blue int64) (int, int, int) {
+	maxValue := maxOfThree(red, green, blue)
+
+	if maxValue > 255 {
+		scaleFactor := float64(255) / float64(maxValue)
+
+		red = int64(float64(red) * scaleFactor)
+		green = int64(float64(green) * scaleFactor)
+		blue = int64(float64(blue) * scaleFactor)
+	}
+
+	return int(red), int(green), int(blue)
+}
+
+func VoteRed() {
+
+	ugnotAmount := std.GetOrigSend().AmountOf("ugnot")
+
+	redVotes = redVotes + ugnotAmount/750
+	red, green, blue := calculateRGB(redVotes, greenVotes, blueVotes)
+	currentColor = ufmt.Sprintf(`rgb(%s, %s, %s)`, strconv.Itoa(red), strconv.Itoa(green), strconv.Itoa(blue))
+}
+
+func VoteGreen() {
+	ugnotAmount := std.GetOrigSend().AmountOf("ugnot")
+
+	greenVotes = greenVotes + ugnotAmount/750
+	red, green, blue := calculateRGB(redVotes, greenVotes, blueVotes)
+	currentColor = ufmt.Sprintf(`rgb(%s, %s, %s)`, strconv.Itoa(red), strconv.Itoa(green), strconv.Itoa(blue))
```

Comment:
Wouldn't `%v` work instead of `strconv`?

Also you could deduplicate these functions quite a bit.

Reply:
Fixed this